clients/include/testing_tbmv_strided_batched.hpp

```diff
@@ -0,0 +1,270 @@
+/* ************************************************************************
```

Comment:
batch_count == 0 ?
we should call the routine with null pointers and having success.

Reply:
I added a test in testing_tbmv_(batched/strided_batched)bad_arg to check that we return success when batch_count == 0 and A and x are nullptrs. This was the easiest way to add them, although I'm not sure they completely fit in the "bad_arg" tests.

Reply:
That is good as I think we are under using bad_arg test, everything possible should be moved in there to reduce the test count.   Otherwise all these bad args cases are multiplied by all the other numeric parameter scans in the yaml.   You get the same code coverage without duplicating test runs by having them in bad_arg.
I don't suggest you do it but many of the invalid_size parameter values could likely be put in bad_arg parameters instead to reduce our test time.

Reply:
batch_count = 0 is not a bad argument. The routine returns success.

Reply:
I don't think it makes any sense to add these tests to the "regular" test set, these are outliers and don't follow the pattern used to generate tests.
If we don't like adding successful tests to the bad_arg set, we could create a new test set, aside from the regular/bad_arg ones. Say, "quick_return" tests, or "nullptr_tests" or something similar? Or even "edge_cases" and add invalid sizes, N == 0 tests, nullptr tests, etc?

Reply:
Make more sense than putting it in bad args :D

Reply:
I will leave it in here for now, adding this should be another PR. I'll start a thread so we can discuss adding another test set and what should be added to it.